hbp_knowledge/receptors/king2015.bel

```diff
@@ -94,250 +94,283 @@ p(HGNC:GNAI1) -> complex(p(HGNC:GNAI1), p(HGNC:CHRNA7))
-
+SET Confidence = "Medium"
+SET MeSHAnatomy = "PC12 Cells"
 p(HGNC:CHRNA7, var("p.345A"), var("p.346A"), var("p.347A"), var("p.348A")) =| complex(p(HGNC:CHRNA7, var("p.345A"), var("p.346A"), var("p.347A"), var("p.348A")), p(HGNC:GNAQ))
 p(HGNC:CHRNA7, var("p.345A"), var("p.346A"), var("p.347A"), var("p.348A")) =| complex(p(HGNC:CHRNA7, var("p.345A"), var("p.346A"), var("p.347A"), var("p.348A")), p(FPLX:"G_beta") ,p(FPLX:"G_gamma"))
+#MeSHAnatomy was set and UNSET
+UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "
 In α7+ cells, choline application resulted in a calcium signal that peaked at
 1050% ΔF/Fθ (±176.4%) in the neurite. Calcium transients in the GC were found to
 last for ∼1.6 s in both α7 and α7+ cells peaking at 1396% (±154.4%) and 1316%
 (±146.9%) ΔF/Fθ, relatively (Fig. 4, A and B)."
-
+SET Confidence = "High"
 SET MeSHAnatomy = "Neurites"
 
-
 composite(p(HGNC:CHRNA7), a(CHEBI:choline)) -> a(CHEBI:"calcium(2+)")
 
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "PC12 cells transfected with α7345–348A showed a reduction in
 choline-mediated calcium responses."
-
+SET Confidence = "High"
 SET MeSHAnatomy = "PC12 Cells"
 
 p(HGNC:CHRNA7, var("p.345A"), var("p.346A"), var("p.347A"), var("p.348A")) -| act(a(CHEBI:choline))
 act(a(CHEBI:choline)) -> a(CHEBI:"calcium(2+)")
 p(HGNC:CHRNA7, var("p.345A"), var("p.346A"), var("p.347A"), var("p.348A")) -| a(CHEBI:"calcium(2+)")
 
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "Most notably, in the GC, the calcium peak values were
 significantly lower in α7345–348A-transfected cells compared with α7 cells after
 Tukey's HSD post hoc comparisons (peak: 741 ± 159.8% ΔF/Fθ, p = 0.006). This
 represents a 46.92% reduction from the α7 baseline calcium response (Fig. 4, A
 and B). This reduction approached significance in the neurite of α7345–348A cells
 (peak: 561 ± 124.9% ΔF/Fθ) (Fig. 4, A and B)"
-
+SET Confidence = "High"
 SET MeSHAnatomy = "Neurites"
 
 composite(p(HGNC:CHRNA7, var("p.345A"), var("p.346A"), var("p.347A"), var("p.348A")), a(CHEBI:choline)) -| a(CHEBI:"calcium(2+)")
 
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "In N2a cells α7345–348A nAChR expression fostered a weak calcium
 response to (10 mM) choline, whereas expression of α7 nAChRs correlated with
 noticeable choline-induced calcium transient. Imaging studies in N2a cells
 expressing α7345–348A nAChRs indicate a significantly lowered calcium signal peak
 to choline (333.4% ΔF/Fθ ± 35.7%) when compared with N2a cells that express the
 α7nAChR (peak: 581% ΔF/Fθ ± 122.7%; p = 0.04) (Fig. 4, D and E)."
-
+SET Confidence = "Medium"
 SET CellLine = "Neuro-2a cell"
 
 composite(p(HGNC:CHRNA7), a(CHEBI:choline)) -> a(CHEBI:"calcium(2+)")
-composite(p(HGNC:CHRNA7, var("p.345A"), var("p.346A"), var("p.347A"), var("p.348A")), a(CHEBI:choline)) -| a(CHEBI:"calcium(2+)")
-
+#composite(p(HGNC:CHRNA7, var("p.345A"), var("p.346A"), var("p.347A"), var("p.348A")), a(CHEBI:choline)) -| a(CHEBI:"calcium(2+)")
```

Comment:
the mutation **significantly** reduces calcium peak, so I would not remove the statement